integration-tests/tests/exit_syscall_payload.rs

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the exit syscall payload test.
+
+#![deny(clippy::all)]
+#![deny(missing_docs)]
+
+mod common;
+
+/// This test runs the exit syscall payload in the SGX keep using the SGX shim.
+#[cfg_attr(not(has_sgx), ignore)]
+#[test]
+fn sys_exit() {
+    let test = common::IntegrationTest::new("test_sys_exit");
+    test.run_test(Some(5), Some(0), None, None, None);
+}
+
+#[cfg_attr(not(has_sgx), ignore)]
+#[test]
+#[should_panic]
+fn timeout_sys_exit() {
+    let test = common::IntegrationTest::new("test_watchdog");
+    test.run_test(Some(5), Some(0), None, None, None);
+}
```

Comment:
We currently have a `watchdog` and `exit_zero` test. We need an `exit_one` test. This is trivial to add, but verifies that changing the input to `exit()` actually has an effect.